### PR TITLE
tiup: refine description about `tiup playground --tag`

### PR DESCRIPTION
**Pull request by hfxsd**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v8.5 (TiDB 8.5 versions)
- [x] v8.4 (TiDB 8.4 versions)
- [x] v8.3 (TiDB 8.3 versions)
- [x] v8.2 (TiDB 8.2 versions)
- [x] v8.1 (TiDB 8.1 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: quick-start-with-tidb.md [hunk 1]
```diff
@@ -68,6 +68,11 @@ As a distributed system, a basic TiDB test cluster usually consists of 2 TiDB in
 
 3. Start the cluster in the current session:
 
+    > **Note:**
+    >
+    > - For the playground operated in the following way, after the test deployment is finished, TiUP will clean up the original cluster data. You will get a new cluster after re-running the command.
+    > - If you want the data to be persisted on storage, run `tiup --tag <your-tag> playground ...`. For details, refer to [Specify a tag when starting the TiDB cluster to store the data](/tiup/tiup-playground.md#specify-a-tag-when-starting-the-tidb-cluster-to-store-the-data).
```
Comment (hfxsd) [inline quick-start-with-tidb.md:74]:
```suggestion
    > - If you want the data to be persisted on storage, run `tiup playground --tag ${tag_name}`. For details, refer to [Specify a tag when starting the TiDB cluster to store the data](/tiup/tiup-playground.md#specify-a-tag-when-starting-the-tidb-cluster-to-store-the-data).
```

File: quick-start-with-tidb.md [hunk 2]
```diff
@@ -189,6 +192,11 @@ As a distributed system, a basic TiDB test cluster usually consists of 2 TiDB in
...
```
Comment (hfxsd) [inline quick-start-with-tidb.md:198]:
```suggestion
    > - If you want the data to be persisted on storage, run `tiup playground --tag ${tag_name}`. For details, refer to [Specify a tag when starting the TiDB cluster to store the data](/tiup/tiup-playground.md#specify-a-tag-when-starting-the-tidb-cluster-to-store-the-data).
```

File: quick-start-with-tidb.md [hunk 3]
```diff
@@ -68,6 +68,11 @@ As a distributed system, a basic TiDB test cluster usually consists of 2 TiDB in
 
 3. Start the cluster in the current session:
 
+    > **Note:**
+    >
+    > - For the playground operated in the following way, after the test deployment is finished, TiUP will clean up the original cluster data. You will get a new cluster after re-running the command.
+    > - If you want the data to be persisted on storage, run `tiup playground --tag ${tag_name}`. For details, refer to [Specify a tag when starting the TiDB cluster to store the data](/tiup/tiup-playground.md#specify-a-tag-when-starting-the-tidb-cluster-to-store-the-data).
```
Comment (lilin90) [inline quick-start-with-tidb.md:74]:
```suggestion
    > For the playground operated in the following way, after the deployment and testing are finished, TiUP will automatically clean up the cluster data. You will get a new cluster after re-running the command. If you want to persist data on storage, then add the `--tag` flag when you start the cluster: `tiup playground --tag ${tag_name}`. For details, see [Specify a tag when starting the TiDB cluster to store the data](/tiup/tiup-playground.md#specify-a-tag-when-starting-the-tidb-cluster-to-store-the-data).
```

File: quick-start-with-tidb.md [hunk 4]
```diff
@@ -189,6 +192,11 @@ As a distributed system, a basic TiDB test cluster usually consists of 2 TiDB in
...
```
Comment (lilin90) [inline quick-start-with-tidb.md:198]:
```suggestion
    > For the playground operated in the following way, after the deployment and testing are finished, TiUP will automatically clean up the cluster data. You will get a new cluster after re-running the command. If you want to persist data on storage, then add the `--tag` flag when starting the cluster: `tiup playground --tag ${tag_name}`. For details, see [Specify a tag when starting the TiDB cluster to store the data](/tiup/tiup-playground.md#specify-a-tag-when-starting-the-tidb-cluster-to-store-the-data).
```

**Review by lilin90 — APPROVED** [review]
/lgtm

**Pull request merged**